e2e/cypress/tests/support/api/setup.ts

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {UserProfile} from '@mattermost/types/users';
```

Comment:
One pain point of reviewing is that ofen, for the support files, it isn't recognized that the old `.js` & `d.ts` files are a rename to `.ts`, so it makes the diffs larger than they really are.